include/rdma/fabric.h

```diff
@@ -163,6 +163,7 @@ typedef struct fid *fid_t;
 #define FI_PEER			(1ULL << 43)
 /* #define FI_XPU_TRIGGER		(1ULL << 44) */
 
+#define FI_TAGGED_DIRECTED_RECV	(1ULL << 46)
```

Comment:
Is it the same bit as FI_TAGGED_MULTI_RECV on purpose...?

Reply:
Definitely not. Thanks for pointing that out.